### DISCUSSION TITLE
[Bug] Auth0 Config on vercel

### DISCUSSION THREAD

**Discussion by nexusdiscord** [lobehub/lobehub#1405] [Self Hosting] [answered]
### 💻 Operating System

Windows

### 📦 Environment

Vercel / Zeabur / Sealos

### 🌐 Browser

Chrome

### 🐛 Bug Description

I followed the steps outlined on https://chat-docs.lobehub.com/en/self-hosting/advanced/authentication.

after clicking the sign in button on the settings page, I got error
![image](https://github.com/lobehub/lobe-chat/assets/63144643/f66886be-56f7-4197-b7b0-fad4ede9f22b)

![image](https://github.com/lobehub/lobe-chat/assets/63144643/9f5c37bb-c9d0-46a8-be0f-ffb435e1cda7)

`    at (node_modules/@auth/core/lib/actions/signin/authorization-url.js:17:23)
    at (node_modules/@auth/core/lib/actions/signin/index.js:10:61)
    at (node_modules/@auth/core/lib/index.js:60:29)
    at (node_modules/@auth/core/index.js:105:28)
    at (node_modules/next/dist/esm/server/future/route-modules/app-route/module.js:189:36)
    at (node_modules/next/dist/esm/server/future/route-modules/app-route/module.js:128:25)
    at (node_modules/next/dist/esm/server/future/route-modules/app-route/module.js:251:29)
    at (node_modules/next/dist/esm/server/web/edge-route-module-wrapper.js:81:20)
    at (node_modules/next/dist/esm/server/web/adapter.js:157:15)`

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
@cy948 please help to check it

**Comment by cy948** [comment]
Please provide the environment variables you used for deployment on Vercel to help us reproduce the error.

**Comment by nexusdiscord** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/63144643/aa4db69d-a694-4f20-8808-ba1b447896c7)

**Comment by cy948** [comment, marked as answer]
`AUTH0_ISSUER` with `https://` prefix? For example: `https://example.auth0.com`

**Reply by cy948** [reply]
![image](https://github.com/lobehub/lobe-chat/assets/67412196/5206d036-3724-44d7-b4cc-1cd7277e975c)


Have you made any changes to the source code? I tried deploying version `0.130.0` with the environment variables shown in the image, and it's running smoothly. 

But if you remove the prefix `https://` of `AUTH0_ISSUER`, it will produce error like this.
![image](https://github.com/lobehub/lobe-chat/assets/67412196/c97ae971-614f-4f54-ab3a-05db93463f16)

[a comment by an automated account is not shown]
